examples/webpack/example-builder.js

```diff
@@ -83,6 +84,32 @@ function getJsSource(chunk, jsName) {
   }
 }
 
+/**
+ * Gets dependencies from the js source.
+ * @param {string} jsSource Source.
+ * @return {Object<string, string>} dependencies
+ */
+function getDependencies(jsSource) {
+  const lines = jsSource.split('\n');
+  const dependencies = {
+    ol: pkg.version
+  };
+  for (let i = 0, ii = lines.length; i < ii; ++i) {
+    const line = lines[i];
+    const importMatch = line.match(importRegEx);
+    if (importMatch) {
+      const imp = importMatch[1];
+      if (!imp.startsWith('ol/') && imp != 'ol') {
+        const dep = imp.split('/')[0];
```

Comment:
If we want to handle scoped packages, this could be something like:
```js
const parts = imp.split('/');
let dep;
if (imp.startsWith('@')) {
  dep = parts.slice(0, 2).join('/');
} else {
  dep = parts[0];
}
```

Reply:
Good suggestion. Done.